daemon/src/rollover/protocol.rs

```diff
@@ -141,3 +190,402 @@ pub(crate) async fn emit_failed(order_id: OrderId, e: anyhow::Error, executor: &
         tracing::error!(%order_id, "Failed to execute rollover failed: {e:#}")
     }
 }
+
+pub(crate) fn build_punish_params(
+    our_role: Role,
+    sk: SecretKey,
+    counterparty_pk: PublicKey,
+    other_msg0: RolloverMsg0,
```

Comment:
can we unify the naming: e.g. 
```suggestion
    counterparty_msg0: RolloverMsg0,
```

or rename `counterparty_pk` to `other_pk`.

Reply:
I opted for `counterparty` instead of `other` because it is more explicit and furhermore we have used it in other places in the code (e.g. the `Dlc`).

See https://github.com/itchysats/itchysats/pull/2248/commits/f7342e12be9941f33eaf735eb8af325d7cf5eaa3